packages/lwc-services/src/utils/webpack/webpack-builder.ts

```diff
@@ -109,6 +104,29 @@ function buildWebpackConfig({ entries, outputDir, moduleDir, mode }: string) {
             }
         }
     }
+
+    if (customConfig) {
+        serverConfig = webpackMerge.smart(serverConfig, customConfig)
```

Comment:
This `merge` doesn't compile because of the [format requirements](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/webpack/index.d.ts#L50).